doc/HOWTO/keys.txt

```diff
@@ -68,5 +68,41 @@ you don't want your key to be protected by a password, remove the flag
     type in the password every time the server needs to access
     the key.
 
+
+4. To generate an EC key
+
+An EC key can be used both for key agreement (ECDH) and signing (ECDSA).
+
+Generating a key for ECC is similar to generating a DSA key, which is also
+a two-step process. First, you have to generate EC parameters from which to
+generate the key:
+
+  openssl ecparam -name prime256v1 -out prime256v1.pem
+
+The prime256v1 (stands for 'X9.62/SECG curve over a 256 bit prime field')
```

Comment:
Not that I consider myself expert in English, but  "*which* stands for ..." would sound better to me. In such case it's probably appropriate to replace parenthesis with commas.